src/braket/ir/jaqcd/program_v1.py

```diff
@@ -153,10 +163,17 @@ class Program(BraketSchemaBase):
         YY,
         Z,
         ZZ
+
+        The type `NoiseInstructions` includes the following instructions:
+        Bit_Flip,
+        Phase_Flip,
+        Depolarizing,
+        Amplitude_Damping,
+        Kraus,
     """
 
     _PROGRAM_HEADER = BraketSchemaHeader(name="braket.ir.jaqcd.program", version="1")
     braketSchemaHeader: BraketSchemaHeader = Field(default=_PROGRAM_HEADER, const=_PROGRAM_HEADER)
-    instructions: List[GateInstructions]
+    instructions: List[Union[GateInstructions, NoiseInstructions]]
     results: Optional[List[Results]]
-    basis_rotation_instructions: Optional[List[GateInstructions]]
+    basis_rotation_instructions: Optional[List[Union[GateInstructions, NoiseInstructions]]]
```

Comment:
+1 same comment as @licedric basis_rotation_instructions are automatically added by the SDK, and NoiseInstructions can't be added to them.